civihr_employee_portal/views/views_export/views_hr_staff_directory.inc

```diff
@@ -334,50 +344,29 @@ $handler->display->display_options['fields']['start_date']['field'] = 'start_dat
 $handler->display->display_options['fields']['start_date']['exclude'] = TRUE;
 $handler->display->display_options['fields']['start_date']['date_format'] = 'long';
 $handler->display->display_options['fields']['start_date']['second_date_format'] = 'long';
-/* Field: HRJobContract Role entity: End_date */
+/* Field: HRJobContract Role entity: Role end date */
 $handler->display->display_options['fields']['end_date_1']['id'] = 'end_date_1';
 $handler->display->display_options['fields']['end_date_1']['table'] = 'hrjc_role';
 $handler->display->display_options['fields']['end_date_1']['field'] = 'role_end_date';
 $handler->display->display_options['fields']['end_date_1']['relationship'] = 'role_jobcontract_id';
 $handler->display->display_options['fields']['end_date_1']['exclude'] = TRUE;
-/* Field: HRJobContract Role entity: Start_date */
+/* Field: Broken/missing handler */
 $handler->display->display_options['fields']['start_date_1']['id'] = 'start_date_1';
 $handler->display->display_options['fields']['start_date_1']['table'] = 'hrjc_role';
 $handler->display->display_options['fields']['start_date_1']['field'] = 'role_star_date';
 $handler->display->display_options['fields']['start_date_1']['relationship'] = 'role_jobcontract_id';
 $handler->display->display_options['fields']['start_date_1']['exclude'] = TRUE;
-/* Field: CiviCRM Contacts: First Name */
-$handler->display->display_options['fields']['first_name_1']['id'] = 'first_name_1';
```

Comment:
Yeah, those were used to try and show the manager's name, but were not associated with the relationship, so they didn't work.  So I just included display_name (so so first, middle and last names could be used to search) and deleted those.